File: Test11/devowel_file.pl
#!/usr/bin/perl -w

undef @result;
open F, '<', "$ARGV[0]" or die $!;
foreach $line (<F>){
	$line =~ s/[aeiou]//gi;
	push @result, $line;
}
close F;

open F, '>', "$ARGV[0]" or die $!;
print F @result;
close F;

File: Test11/devowel_file.py
#!/usr/bin/python3

import sys, re

with open(sys.argv[1], 'r') as f:
	data = f.read()
data = re.sub(r'[aeiou]', '', data, flags = re.I)

f=open(sys.argv[1], 'w')
f.write(data)
f.close()
